src/arguments_parser.star

```diff
@@ -3,13 +3,41 @@ system_variables = import_module("github.com/logos-co/wakurtosis/src/system_vari
 
 
 def apply_default_to_input_args(input_args):
```

Comment:
Hmm. Now that we have a decent amount of parameters, maybe it is nice to use [read_file](https://docs.kurtosis.com/reference/starlark-instructions#read_file) and read a file where all these parameters are set. A json can be nice since we can encode/decode it with Starlark. 

You can do this in this PR, or in a different issue, do as you like.

Either way, I think this will be necessary, and I see two options:
- We force the user to set up specific mandatory parameters in the json, raising an error if they are not set.
- We automatically set parameters (like I was doing with my only two parameters) but for all of them.

Personally, I think the best choice is to do the 1st option. What do you think?

Reply:
Agree, and there will be even more. I ll do it in another PR.